src/node/coin.cpp

```diff
@@ -23,4 +23,23 @@ void FindCoins(const NodeContext& node, std::map<COutPoint, Coin>& coins)
         }
     }
 }
+
+std::vector<Coin> FindCoins(const node::NodeContext& node, std::vector<COutPoint>& outpoints)
```

Comment:
In ccbc0044cbe16b52210eebe700b0865cc2ac1946:

`outpoints` should be a `const` reference because you don't ever modify it. Without `const` this function is very confusing to read.

Reply:
Fixed, Thanks!